src/core/utils/ifSpecBuilder.cpp

```diff
@@ -76,12 +76,14 @@ std::size_t CStringIdListSpecBuilder::calcStorageSize() const {
   return cmStaticList ? 0 : mDynamicList.size() * sizeof(CStringDictionary::TStringId);
 }
 
-std::tuple<const CStringDictionary::TStringId*, TForteUInt8> CStringIdListSpecBuilder::build(CMixedStorage &paStorage) const {
+template<typename TSize>
```

Comment:
Hi.
Not sure if `TSize` (Type Size, which is also a bit strange as T indicates a type definition, which you don't do here) is a good name here, as its not really a size but some sort of index or numeric identifier (`TPortID`, `TEventID`).
I personally would go with a generic `Type` or `IndexType`

Reply:
@azoitl what is your opinion here?

Reply:
@MartinMelikMerkumians I agree IndexType would be better